### DISCUSSION TITLE
Pikka Queries

### DISCUSSION THREAD

**Discussion by csmltd** [rsayfutdinov/Pikka#23] [General]
Hi Rusian

Can you please confirm the following -

1. If I purchase a license how many machines can this be used on?
2. I do not use iCloud - can Pikka be synced to other cloud storage options - e.g Dropbox
3. Is it possible to backup Pikka configuration settings, palettes etc if I wish to create my own backups? Where are these settings to be found?

Regards

**Comment by rsayfutdinov** [comment]
> Hi Rusian
> 
> 
> 
> Can you please confirm the following -
> 
> 
> 
> 1. If I purchase a license how many machines can this be used on?
> 
> 2. I do not use iCloud - can Pikka be synced to other cloud storage options - e.g Dropbox
> 
> 3. Is it possible to backup Pikka configuration settings, palettes etc if I wish to create my own backups? Where are these settings to be found?
> 
> 
> 
> Regards

Hi,

1. License is not limited. You can use it on any machine you have. 
2. At the Moment is only iCloud sync possible.
3. Not really. You can export palettes in different formats including “apple palette format(clr)”. I’m already working on importing .clr files into the app. So you will be able to use it as some kind of “backup” soon.

Regards,
Ruslan

**Comment by chadcribbins** [comment]
There was a 7 day trial? 
Id like to cancel my trail, but i can't see where that is?

**Reply by mijnnaamisramon** [reply]
Same here.... it asked to confirm a subscription, but I can't see it (and thus cancel it) in my subscriptions. How do I cancel my trial?

**Reply by mijnnaamisramon** [reply]
Ah, Apple already assigned a refund. So I think this is closed for me. :)

**Reply by rsayfutdinov** [reply]
I'm sorry, I wasn't able to reply right away. Yes, you can always refund a purchase through the app store.

**Reply by mijnnaamisramon** [reply]
No worries. But you might wanna look into this, you can't start a trial without accepting the lifetime subscription.... seems unnecessary...

**Reply by mark5422** [reply]
On this 7-day trial, when I first opened the app and tried to sign up for it from the App Store, when I was selecting lifetime it tried to charge me for the lifetime subscription without giving me a trial as expected. The trial on the other options was only 3 days and it never mentioned that anywhere. Also once completing purchase I was left with the purchase window like nothing had happened.

**Reply by rsayfutdinov** [reply]
Hey mark5422,
Thanks for letting me know, that the purchase process did not work as it should. 

Lifetime purchase doesn't offer any trials(Appstore limitation). And actually, you don't need to start a subscription to test the App. Pikka provides a lot of features for free.

**Comment by apjms** [comment]
Hello—I"ve just migrated to a new laptop. When I enter the license code for Pikka, I get the message that the code is already in use. It seems it's bound to another machine, presumably my previous laptop. I read above that it should not be a problem as it can be used on multiple machines, yet this is the message I get. Can you tell me what to do to get it working on my new laptop?
